registry/domain/models/organization.py

```diff
@@ -174,17 +179,23 @@ def publish_to_ipfs(self):
         self.__metadata_ipfs_uri = f"ipfs://{ipfs_hash}"
 
     def setup_id(self):
-        if self.is_org_uuid_set():
-            self.__uuid = uuid4().hex
-        if self.org_type == "individual" and self.is_org_id_set():
+        if self.__org_type == OrganizationType.INDIVIDUAL.value:
```

Comment:
self.__org_type == OrganizationType.INDIVIDUAL.value
lowercase/uppercase the values and then check.
type should be case insensitive